_editorial_policies/04_publication_ethics.md

```diff
@@ -0,0 +1,51 @@
+- The sole responsibility for acceptance or rejection of a manuscript rests with the assigned editor. Responsible and prudent exercise of this duty normally requires that the editor seek advice from reviewers, chosen for their expertise and good judgment, as to the quality and reliability of manuscripts submitted for publication. Editors should communicate openly with authors and are discouraged from providing comments under the guise of anonymous review. However, manuscripts may be rejected without external review if considered by the editors to be inappropriate for the journal. The presubmission letter process is the primary manner that such rejections are avoided, but editors reserve the right to make such rejections. Such rejections may be based on the failure of the manuscript to fit the scope of the journal, to be of current or sufficiently broad interest for the readership of the journal, to provide adequate depth of content, to be written in acceptable English, or other reasons.
+- When an editor with decision-making responsibility decides to provide scientific or other comments on work described in an assigned manuscript, they should do this transparently within the communication of their editorial decision and not under the guise of an anonymous peer reviewer. 
+- Editors should not disclose any information about a manuscript under consideration to anyone other than those from whom professional advice is sought. After a decision has been made about a manuscript, but before publication, the editor may disclose or publish manuscript titles and authors’ names of papers that have been accepted for publication, but no more than that unless the author’s permission has been obtained. If a decision has been made to reject a manuscript for ethical violations, the editor may disclose the manuscript title and authors' names to other journal editors.
+- An editor should respect the intellectual independence of authors.
+- Editorial responsibility and authority for any manuscript authored by an editor and submitted to the editor’s journal should be delegated to some other qualified person, such as another editor of that journal / member of its Editorial Advisory Board. Editors should also delegate manuscripts submitted by recent graduates, or by colleagues at the same institution, to some other qualified editor. A collaboration between the author and editor, which is separate from the paper under review, or may may not represent a conflict requiring manuscript delegation. However, an editor should not be responsible for a paper which would directly further their own research program; for example, an editor should not manage a paper that will serve as preliminary results for a grant or grant renewal that the editor would financially benefit from, even if they were not involved in writing the grant.  Editors should review the LiveCoMS bylaws for more information on what constitutes a conflict. 
```

Comment:
This brought up a question if the conflicts is too lenient.  Specifically if a collaboration is automatically a conflict, and it defines more precisely what definitely would be a conflict.  The issue since so many editors are working on reviews, that we will run out of editors that can manage an article.  But since we have enough editors now, maybe this isn't a big deal to fix.

Maybe it should be changed to the same conflict for reviewers - a PUBLICATION isn't necessarily disqualifying if it's a large multiauthor publication that the two people didn't directly work on together.

Reply:
I'm unclear on what you're saying here in your comment. Are you saying that this provides a more precise definition than our current one and is more helpful, so we should move to this? Or are you saying you want further refinement of this, and if so, how?

Reply:
@dmzuckerman originally said that "collaborator" should automatically be a conflict, I was trying to say it might be a conflict.

Reply:
OK. Suggest "or may may not" -> "may or may not"